TestMethod.sc

```diff
@@ -25,4 +25,43 @@ TestMethod : UnitTest {
 		);
 	}
 
-} // End class
\ No newline at end of file
+	test_classnameAsMethodCrash {
+	    try
+	    {	    
+	        1 fooBar: 2;
```

Comment:
btw. it is a good style (just my view) to leave out the semicolon for the last (return) statement in a function: this shows that nothing could come after it and is also simpler. 

No need to correct unless you feel like it.